.github/workflows/node-zxc-build-release-artifact.yaml

```diff
@@ -58,6 +58,11 @@ on:
         type: string
         required: false
         default: "Artifact"
+      generate-artifacts:
+        description: "Generate Artifacts:"
+        type: boolean
+        required: false
+        default: true
```

Comment:
This forces almost every step to be conditional and is undesirable. Instead we should split the previewnet release out to separate reusable workflow.